src/ui/opengl/graphics.cpp

```diff
@@ -0,0 +1,206 @@
+#include "ui/opengl/graphics.h"
+
+using std::vector;
+
+using delaunay::LongInt;
+using delaunay::PointSetArray;
+
+
+
+void drawAPoint(double x, double y) {
+	glPointSize(5);
+	glBegin(GL_POINTS);
+	glColor3f(0,0,0);
+		glVertex2d(x,y);
+	glEnd();
+	glPointSize(1);
+}
+
+
+
+void drawALine(double x1, double y1, double x2, double y2) {
+	glPointSize(1);
+	glBegin(GL_LINE_LOOP);
+	glColor3f(0,0,1);
+		glVertex2d(x1,y1);
+		glVertex2d(x2,y2);
+	glEnd();
+	glPointSize(1);
+}
+
+
+
+void drawATriangle(double x1, double y1, double x2, double y2, double x3, double y3) {
+	glBegin(GL_POLYGON);
+	glColor3f(0,0.5,0);
+		glVertex2d(x1,y1);
+		glVertex2d(x2,y2);
+		glVertex2d(x3,y3);
+	glEnd();
+}
+
+
+
+void drawPointSetArray(const PointSetArray<LongInt>& pointSet) {
+	// Draw input points
+	for (int i = 1; i <= pointSet.noPt(); i++){
+		LongInt px, py;
+		pointSet.getPoint(i, px, py);
+		drawAPoint(px.doubleValue(), py.doubleValue());
+	}
+}
+
+
+
+// void drawDelaunayStuff() {
```

Comment:
Could either remove this, or make use of it. But leaving it commented out is stupid.